new-lamassu-admin/src/pages/Blacklist/Blacklist.js

```diff
@@ -108,6 +152,8 @@ const Blacklist = () => {
 
   const rejectAddressReuse = complianceConfig?.rejectAddressReuse ?? false
 
+  const enablePaperWalletOnly = !!complianceConfig?.enablePaperWalletOnly
```

Comment:
You can change the variable on line 153 to match the `!!` format of this new one. For consistency's sake